apps/src/componentLibrary/slider/slider.module.scss

```diff
@@ -11,6 +11,9 @@ $slider-brand-track-disabled-color: $light_gray_100;
 $slider-white-track-fill-color: $light_white;
 $slider-white-track-empty-color: $light_gray_400;
 $slider-white-track-disabled-color: $light_gray_900;
+$slider-aqua-track-fill-color: $brand_aqua_50;
```

Comment:
please use primitive colors here.

Sorry for confusion, by the time `Slider` was implemented we indeed were using `shared/color.scss`, however now after discussing moving dsco to separate package we agreed to use primitive colors instead. (this will allow us to decouple from code-dot-org repo and will make it easier to keep colors in sync with figma)

Reply:
Will do.